src/compare_vocabs.py

```diff
@@ -25,7 +26,12 @@ def save_json(data: dict, path: Path):
 def load_concept_csv(filepath: Path) -> pd.DataFrame:
     """Load Athena concept table as a DataFrame"""
     return pd.read_csv(filepath, sep="\t", dtype=str, keep_default_na=False)
-    
+
+
+def count_terms(terms: dict) -> int:
+    """Print the number of terms in the terms JSON"""
```

Comment:
```suggestion
    """count the number of terms in the terms JSON"""
```
not sure this really needs to be a function tbh